spec/factories/proposals.rb

```diff
@@ -5,6 +5,7 @@
     abstract "This and that"
     details "Various other things"
     pitch "Baseball."
+    track { Track.first || FactoryGirl.create(:track) }
```

Comment:
Not sure how I feel about track being set on a factory proposal in all cases. Maybe have a trait :with_track for this instead? Or just pass a track in for those tests where you need it.